donut/modules/uploads/helpers.py

```diff
@@ -0,0 +1,75 @@
+import flask
+import os
+from flask import current_app
+import glob
+
+ALLOWED_EXTENSIONS = set(
+    ['docx', 'doc', 'txt', 'pdf', 'png', 'jpg', 'jpeg', 'gif'])
```

Comment:
Why is this a set?

Reply:
fast(er) lookup for sets, can't have duplicates :D